specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -378,63 +674,360 @@
+          "$ref": "#/definitions/TranscriptionPiiTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "TranscriptionPii"
+    },
+    "TranscriptionPiiConversationItemResult": {
+      "type": "object",
+      "required": [
+        "id",
+        "redactedContent"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "redactedContent": {
+          "type": "object",
+          "$ref": "#/definitions/RedactedTranscriptContent"
+        },
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Entity"
+          }
+        }
+      }
+    },
+    "AudioTiming": {
+      "type": "object",
+      "properties": {
+        "offset": {
+          "description": "Offset from start of speech audio, in ticks.",
+          "type": "integer"
+        },
+        "duration": {
+          "description": "Duration of word articulation, in ticks.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "AnalyzeConversationTranscriptionPiiLROResult": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROResult"
+        }
+      ],
+      "properties": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TranscriptionPiiResult"
+          }
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "TranscriptionPiiResults"
+    },
+    "TranscriptionPiiResult": {
+      "required": [
+        "conversationItems"
+      ],
+      "properties": {
+        "conversationItems": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TranscriptionPiiConversationItemResult"
+          }
+        },
+        "allOf": {
+          "$ref": "#/definitions/ConversationResult"
+        }
+      }
+    },
+    "TranscriptionPiiTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Transcription PII task.",
+      "properties": {
+        "piiCategories": {
+          "$ref": "#/definitions/TranscriptionPiiCategories"
+        },
+        "includeAudioRedaction": {
+          "type": "boolean",
+          "default": false
+        },
+        "showDetectedEntityTypes": {
+          "type": "boolean",
+          "default": false
+        },
+        "redactionSource":{
+          "type": "string",
+          "$ref": "#/definitions/ContentSource"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/TaskParameters"
+        },
+        {
+          "$ref": "#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "ContentSource": {
+      "type": "string",
+      "description": "Enumeration of supported content source types.",
```

Comment:
It would be really good to elaborate on the values here, or point to STT documentation maybe.